test_config.yaml

```diff
@@ -35,23 +35,17 @@
     - proteomics_tmt:
         files:
           - name: agora_proteomics_tmt
-            id: syn32188234
+            id: syn35221005
             format: csv
         final_format: json
         provenance:
-          - syn32188234
+          - syn35221005
         column_rename:
-          peptide: uniqid
-          geneid: hgnc_symbol
           protid: uniprotid
           ci_l: ci_lwr
           ci_h: ci_upr
-          ensg: ensemble_gene_id
           coefficient: log2_fc
           fdr_pval: cor_pval
```

Comment:
These column renames are still not quite right, we need to rename only hgnc and ensg:

        column_rename:
          genename: hgnc_symbol
          ensg: ensemble_gene_id